### PR TITLE
Add interrupt pending feature to deployments

### PR DESCRIPTION
**Pull request by schwitzguebel**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

**Review by danielthirion — DISMISSED** [review]

File: docker-compose/nginx/nginx.conf [hunk 1]
```diff
@@ -19,6 +19,9 @@ http {
     upstream config-notification-server-upstream {
         server config-notification-server:8080;
     }
+    upstream interruption-server-upstream {
+            server interruption-server:8080;
```
Comment (vbochetRTE) [inline docker-compose/nginx/nginx.conf:23]: Reduce indentation

File: docker-compose/nginx/nginx.conf [hunk 2]
```diff
@@ -19,6 +19,9 @@ http {
     upstream config-notification-server-upstream {
         server config-notification-server:8080;
     }
+    upstream interruption-server-upstream {
```
Comment (vbochetRTE) [inline docker-compose/nginx/nginx.conf:22]: You may add the same in nginx-dev

File: k8s/bases/common/gridcapa-interruption-server-configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: interruption-server
```
Comment (vbochetRTE) [inline k8s/bases/common/gridcapa-interruption-server-configmap.yaml:4]: Almost all configmap names end with "-configmap". Rao-runner's configmap is an exception.
You should apply the rule here.

File: k8s/bases/cse-export-d2cc/cse-export-d2cc-job-launcher-configmap.yaml [hunk 4]
```diff
@@ -51,3 +51,4 @@ data:
       url:
         task-manager-timestamp-url: http://cse-export-d2cc-task-manager/tasks/
         task-manager-business-date-url: http://cse-export-d2cc-task-manager/tasks/businessdate/
+        interrupt-run-url: http://interruption-server/interrupt/
```
Comment (vbochetRTE) [inline k8s/bases/cse-export-d2cc/cse-export-d2cc-job-launcher-configmap.yaml:54]: Is there nothing to change in the Ingress configurations to make the interruption server reachable from other containers?

File: docker-compose/cse-export-d2cc/config/cse-export-d2cc-job-launcher-application.yml [hunk 5]
```diff
@@ -40,3 +40,4 @@ job-launcher:
   url:
     task-manager-timestamp-url: http://cse-export-d2cc-task-manager:8080/tasks/
     task-manager-business-date-url: http://cse-export-d2cc-task-manager:8080/tasks/businessdate/
+    interrupt-run-url: http://interruption-server:8080/interrupt/
```
[the remark this answers is not in the document]
Comment (schwitzguebel) [reply docker-compose/cse-export-d2cc/config/cse-export-d2cc-job-launcher-application.yml:43]: good point

File: docker-compose/nginx/nginx.conf (hunk 2 above)
Comment (schwitzguebel) [reply docker-compose/nginx/nginx.conf:22]: ok

File: k8s/bases/cse-export-d2cc/cse-export-d2cc-job-launcher-configmap.yaml (hunk 4 above)
Comment (schwitzguebel) [reply k8s/bases/cse-export-d2cc/cse-export-d2cc-job-launcher-configmap.yaml:54]: not that I noticed during azure testing

**Review by vbochetRTE — APPROVED** [review]

**Pull request merged**